<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Battleship</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="FieldView" name="fieldCurrentPlayer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="FieldView" name="fieldOtherPlayer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelP1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelP2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readyButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewGame"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewGame">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FieldView</class>
   <extends>QGraphicsView</extends>
   <header>fieldview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
